samples/bluetooth/central_hr_coded/README.rst

```diff
@@ -26,6 +26,11 @@ The sample supports the following development kits:
    :header: heading
    :rows: nrf5340dk_nrf5340_cpuapp_and_cpuappns, nrf52840dongle_nrf52840, nrf52840dk_nrf52840
 
+.. note::
```

Comment:
It would be better if this note could be added as part of an include, instead of copy-pasting the same note to every sample that supports nrf53.

Reply:
Maybe a link to this page: http://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_multi_image.html#image-specific-variables

Reply:
There is currently link to the http://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_multi_image.html#image-specific-variables. I could create for example a hci_rmpsg_overlay.txt file and inlude it in those samples.

Reply:
I moved it to the generic hci_rpmsg_overlay.txt and include it in every sample which require additional configuration for the hci_rpmsg.